### ISSUE TITLE
[clsim] frames missing at end of file (Trac #1464)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1456]
There are some times where clsim will not process a few of the last frames in an input file.  This seems to be a result of insufficient cleanup of the frame buffers in the `Finish()` method.  Suggested solution is to loop over the cleanup code until the buffers are actually empty.

Migrated from https://code.icecube.wisc.edu/ticket/1464
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "There are some times where clsim will not process a few of the last frames in an input file.  This seems to be a result of insufficient cleanup of the frame buffers in the `Finish()` method.  Suggested solution is to loop over the cleanup code until the buffers are actually empty.", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "[clsim] frames missing at end of file", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-12-03T17:14:32", 
    "milestone": "", 
    "owner": "claudio.kopper", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151203T17:31:52: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 140057:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140057"
> really make sure the buffers are empty on finish. closes #1456
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas commented:

> Milestone IceSim 5.0 deleted


* olivas changed milestone from "IceSim 5.0" to ""
